### PR TITLE
Doc rework

### PR DESCRIPTION
**Pull request by LillyJekow**
Reworked the entire documentation

### CODE REVIEW

**Comment by LaurentMox** [comment]
@LillyJekow Can you please rebase on master?

**Review by germainruget — APPROVED** [review]
This looks good to me, it looks way more clear than what we had before

File: docs/partner/index.md [hunk 1]
```diff
@@ -1,48 +1,68 @@
...
- - The email domain does not match the organization URL provided
- - The project is not possible with the LumApps marketplace
+Once you are enrolled in the developer program, you need to create a partner organization. A partner organization is your identity on the LumApps Marketplace. The partner organization is listed as the owner of the extensions you create.
 
-## Manage my partner organization
-Your partner information are available in the LumApps extension playground, in the back office interface.
-As for now you cannot edit your partner information by yourself. You need to contact the LumApps marketplace support if you want to edit any of your partner organization information.   
+There are two steps to register your partner organization:
+
+-   Create your private GitHub repository dedicated your marketplace extensions.
```
Comment (gregorycat) [inline docs/partner/index.md:35]: With the new Developer portal this part has been removed, we do not create a GitHub repository anymore

File: docs/partner/index.md [hunk 2]
```diff
@@ -1,48 +1,68 @@
...
+For the GitHub repository, we need several pieces of information:
+
+| Information        | Description                                                               |
+\| ------------------ \| ------------------------------------------------------------------------- \|
+| Organization name  | The name your want to give your partner organization.  We recommend that you use the name of your company.     |
+| Developer name     | First and last name of the main (or first) developer for the marketplace extensions. |
+| Developer email    | Email address of the main (or first) developer for the marketplace extensions.       |
+| GitHub login       | The GitHub identity of the main developer for the Marketplace extensions. |
+
+>**Note:** You will be able to add more developers later on.
```
Comment (gregorycat) [inline docs/partner/index.md:47]: Same here, no more GitHub with the developer portal

**Review by gregorycat — CHANGES_REQUESTED** [review]

**Review by germainruget — APPROVED** [review]

**Review by germainruget — APPROVED** [review]

**Pull request merged**